opal/corelib/string/encoding.rb

```diff
@@ -103,6 +103,7 @@ def bytesize(*)
 
   class ::EncodingError < ::StandardError; end
   class ::CompatibilityError < ::EncodingError; end
+  class UndefinedConversionError < ::EncodingError; end
```

Comment:
Can you please add the double column here? It makes the lookup faster.

Reply:
That wouldn't be correct, it must be Encoding::UndefinedConversionError, see https://github.com/opal/opal/blob/89661b5226079ebf20230412c7c3360108d8a3ae/lib/opal/source_map/map.rb#L14

Reply:
Oh, right!